### ISSUE TITLE
King's Quest 2 - Romancing the Throne (Atari ST)

### ISSUE THREAD

**Issue by Python-Exoproject** [exoscoriae/eXoScummVM#51]
Cant find a copy that will successfully extract with my tools. There are two versions with stx extension, both giving an error when reading the second disk. There is also a cracked one disk version but it only has a .prg and .img on it which is not usefull for ScummVM

**Comment by exoscoriae** [comment]
Have you tried STEEM with the PASTI extension? This allows it to handle STX images. If you have trouble, post the disk image here and I'll do it.

**Comment by Python-Exoproject** [comment]
Yeah, STEEM is how I extracted the first disk but it didnt like any copy of the second disk I had found. 

We either need:

1. A different dump of the game which has no error on the second disk
2. A way to use the 1 disk version with a .prg and .img
3. Someone to have better luck than me with 1. above

This is the error the second disk gives in STEEM when trying to access the drive its mounted on:
![image](https://user-images.githubusercontent.com/67406191/117307034-906f4880-aec3-11eb-8921-56728cc82ed8.png)

The image came from http://www.atarimania.com/game-atari-st-king-s-quest-ii-romancing-the-throne_9748.html
